index.js

```diff
@@ -101,14 +112,5 @@ TypeScript.prototype.cacheKeyProcessString = function(string, relativePath) {
 };
 
 TypeScript.prototype.processString = function (string, relativePath) {
-  try{
-    return ts.transpileModule(string, {compilerOptions: this.options, fileName: relativePath}).outputText;
-  }catch(e){
-    console.error('TYPESCRIPT ERROR:');
-    console.error(e.stack + '\n');
-
-    return '';
-  }
```

Comment:
I can't remember the details, but I believe it the typescript compiler would throw errors quite liberally in cases where it doesn't matter (empty file, .d.ts or something like that). Anyhow, if you tested that it works for glimmer then it's probably fine to remove!